tutorials/text/annotations.py

```diff
@@ -47,22 +55,45 @@
 # 'data'              use the axes data coordinate system
 # ==================  ========================================================
 #
-# For example to place the text coordinates in fractional axes
-# coordinates, one could do::
+# The following strings are also valid arguments for *textcoords*
 #
-#     ax.annotate('local max', xy=(3, 1),  xycoords='data',
-#                 xytext=(0.8, 0.95), textcoords='axes fraction',
-#                 arrowprops=dict(facecolor='black', shrink=0.05),
-#                 horizontalalignment='right', verticalalignment='top',
-#                 )
+# ==================  ========================================================
+# argument            coordinate system
+# ==================  ========================================================
+# 'offset points'     offset (in points) from the xy value
+# 'offset pixels'     offset (in pixels) from the xy value
+# ==================  ========================================================
 #
 # For physical coordinate systems (points or pixels) the origin is the
-# bottom-left of the figure or axes.
+# bottom-left of the figure or axes. Points are
+# `typographic points <https://en.wikipedia.org/wiki/Point_(typography)>`_
+# meaning that they are a physical unit measuring 1/72 of an inch. Points and
+# pixels are discussed in further detail in :ref:`transforms-fig-scale-dpi`.
 #
-# Optionally, you can enable drawing of an arrow from the text to the annotated
-# point by giving a dictionary of arrow properties in the optional keyword
-# argument *arrowprops*.
+# .. _annotation-data:
+#
+# Annotating data
+# ~~~~~~~~~~~~~~~
+#
+# This example places the text coordinates in fractional axes coordinates:
+
+fig, ax = plt.subplots(figsize=(3, 3))
+ax.scatter(3, 1, s=20)
+ax.annotate('local max',
+            xy=(3, 1),  xycoords='data',
+            xytext=(0.8, 0.95), textcoords='axes fraction',
+            horizontalalignment='right', verticalalignment='top')
```

Comment:
I think this is not very instructive, because the example is not quite realistic:

![grafik](https://user-images.githubusercontent.com/2836374/198889981-6e31a4db-8ddf-412c-b64a-7ae9a103b156.png)

IMHO one rarely would want to place the label in different coordinate systems when one does not have a connecting arrow, because the label can be very far away from the data. Also the label name "local max" does not make sense here.

Suggestion: Take the previous example and place the label in the top-left corner.

Reply:
I don't see a new / updated commit. Have you forgotten to push?